packages/fast-components-styles-msft/src/heading/index.ts

```diff
@@ -0,0 +1,166 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { IHeadingClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { toPx } from "../utilities/units";
+import { applyBreakpoint, breakpoints } from "../utilities/breakpoints";
+
+function applyHeadingStyles(): ICSSRules<IDesignSystem> {
+    return {
+        fontWeight: "700"
+    };
+}
+
+const styles: ComponentStyles<IHeadingClassNameContract, IDesignSystem> = {
+    heading_1: {
+        padding: `${toPx(37)} ${toPx(0)} ${toPx(3)}`,
```

Comment:
If the values are known I don't think you really need to generate the value at runtime - you may as well just use "37px 0px 3px"

Reply:
These are going away.